dev-python/doit/doit-0.33.1.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Any chancce of 9?

Reply:
Need python/configclass[python_targets_python3_9] for that:

```
   dev-python/doit-py/doit-py-0.5.0.ebuild: BDEPEND: amd64(default/linux/amd64/17.0)
[     'dev-python/configclass[python_targets_python3_6(-)?,python_targets_python3_7(-)?,python_targets_python3_8(-)?,python_targets_python3_9(-)?,-python_single_target_python3_6(-),-python_single_target_python3_7(-),-python_single_target_python3_8(-),-python_single_target_python3_9(-)]']
```

Reply:
Fancy adding it, if it works & tests pass there too?

Reply:
(This isn't a blocker for merging this, still on queue to test.)

Reply:
There more and more dependency broken - dev-python/configclass requires dev-python/mergedict

Reply:
Well, someone needs to do it, but I understand that it's not something you have to do either, if you get me. 👍